src/TodoInput.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function TodoInput({
+  value,
+  placeholder,
+  onChange,
+  onClick,
+}) {
+  return (
+    <div>
+      <input type="text" id="todo" value={value} placeholder={placeholder} onChange={(e) => onChange(e.target.value)} />
+      <button type="button" onClick={() => onClick()}>추가</button>
```

Comment:
```suggestion
      <input 
        type="text" 
        value={value} 
        placeholder={placeholder} 
        onChange={onChange}
      />
      <button type="button" onClick={onClick}>
        추가
      </button>
```

1. 가독성을 위해 적절히 줄바꿈은 해주는 게 좋습니다.
2. 핸들러를 전달하는 방식은 고정적이지 않고 상황마다 적절한 형태가 있습니다.

또한 input에 id는 필요하지 않습니다:)

- https://stackoverflow.com/questions/1397592/difference-between-id-and-name-attributes-in-html

Reply:
코드는 가독성이 중요하죠 : )
적절한 줄바꿈 지향하도록 하겠습니다~